compiler/rustc_ast_lowering/src/item.rs

```diff
@@ -1379,58 +1379,66 @@ impl<'hir> LoweringContext<'_, 'hir> {
         }
     }
 
+    // Get the `LocalId` of `bound_pred.bounded_ty`, but only if it's a plain type parameter.
+    fn get_def_id(&mut self, bound_pred: &WhereBoundPredicate) -> Option<LocalDefId> {
```

Comment:
Can you give a longer name to this function? There are already plenty of ways to get def_ids here.
Perhaps `predicate_id`?

Reply:
I think `ty_param_id` is probably more accurate.